src/index.html

```diff
@@ -9,6 +9,68 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="container__logo">
+        <img 
+          src="images/logo.png" 
+          alt="moyo-logo" 
+          class="header__img"
+        >
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="#apple" class="is-active nav__a">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#samsung" class="nav__a">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__li">
```

Comment:
add empty line according to [this](https://github.com/mate-academy/layout_moyo-header/pull/2680#discussion_r1112691703) remark 
```suggestion
          </li>
          <li class="nav__li">
```

Reply:
I did it as I understood it)